client/app/assets/styles/variables.js

```diff
@@ -113,4 +123,24 @@ module.exports = {
   '--SearchBar_mobileTriangleSize': '8px',
   '--SearchBar_iconSize': '16px',
   '--SearchBar_iconTopMarginFix': '4px',
+
+  '--MenuItem_backgroundColorHover': backgroundLightColorHover,
+  '--MenuItem_borderColor': borderColor,
+  '--MenuItem_paddingTopbarVertical': paddingTopbarVertical,
+  '--MenuItem_paddingTopbarHorizontal': paddingTopbarHorizontal,
+  '--MenuItem_textColor': textColor,
+  '--MenuItem_textColorFocus': textColorFocus,
+  '--MenuItem_letterSpacing': '0.09px',
+
+  '--Menu_fontSize': fontSize,
+  '--Menu_fontSizeSmall': fontSizeSmall,
+  '--Menu_textColor': textColor,
+  '--Menu_textColorFocus': textColorFocus,
+  '--Menu_colorBackground': backgroundLightColor,
+  '--Menu_borderColor': borderColor,
+  '--Menu_boxShadow': '0px 2px 4px 0px rgba(0, 0, 0, 0.1)',
+  '--Menu_iconPadding': '0.7em',
+  '--Menu_zIndex': '1',
+  '--Menu_labelPaddingVertical': '1.28571429rem',
+  '--Menu_labelPaddingHorizontal': '1.7142857rem',
```

Comment:
Do the rem units work with the main application where we cannot control the root font size?